Source/monstdat.cpp

```diff
@@ -23,7 +24,7 @@ constexpr uint16_t Uniq(_unique_items item)
 } // namespace
 
 /** Contains the data related to each monster ID. */
-const MonsterData MonstersData[] = {
+MonsterData MonstersData[] = {

```

Comment:
Manipulating the table during runtime doesn't sound like a good idea :thinking:

Reply:
Also should use MonstersDataSize to specify the size if you are adding it so they don't go out of sync.